ui/packages/gallery/src/Gallery.svelte

```diff
@@ -87,7 +91,15 @@
 
 	$: can_zoom = window_height >= height;
 
-	$: ({ styles } = get_styles(style, ["grid"]));
+	function add_height_to_styles(style: Styles) {
+		styles = get_styles(style, ["grid"]).styles;
```

Comment:
Adding `height` to the array passed to `get_styles` only adds height to the css if height=auto which is not what we want in this case.